### PR TITLE
[post] skipper

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Ref https://github.com/symplify/symplify/issues/2618

**Comment by TomasVotruba** [comment]
You motivated me to write a post about Skipper :smiley: 

Here it is!
Could I ask you for a review here? So people can upgrade easily

**Review by tomasnorre — COMMENTED** [review]
I only have a few comments in the beginning, but nothing to the content itself. 
This post will help people understand, use and update to the new format.

File: packages/blog/data/2020/2020-12-10-new-in-symplify-9-skipper-skipping-files-and-rules-made-simple.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+---
+id: 292
+title: "New in Symplify 9: Skipper - Skipping Files and Rules made Simple"
+perex: |
+    Symplify 9 brings **another component to easy life for package developers**. In Rector and ECS, you can ignore specific absolute paths or dynamic with `fnmatch()`. You can also ignore specific Rector, Fixer, and Sniff classes.
+    <br><br>
+    Both packages used almost the same syntax, yet there were differences based on which of them got fixed first. New Skipper component prevents this and allows **you to use standardized syntax in single `skip` parameters**.
```
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply packages/blog/data/2020/2020-12-10-new-in-symplify-9-skipper-skipping-files-and-rules-made-simple.md:7]: It is meant to be in the past, i.e. Symplify 8.
I'll try to rephrase to make it more clear.

File: packages/blog/data/2020/2020-12-10-new-in-symplify-9-skipper-skipping-files-and-rules-made-simple.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+tweet: "New Post on #php 🐘 blog: New in #Symplify 9: Skipper -
+Skipping Files and Rules made Simple"
+tweet_image: "/assets/images/posts/2020/skipper-kangaroo.jpg"
+---
+
+When I started writing this post, I remembered a TV series from my childhood about the hero kangaroo:
+
+<img src="/assets/images/posts/2020/skipper-kangaroo.jpg" class="img-thumbnail">
```
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply packages/blog/data/2020/2020-12-10-new-in-symplify-9-skipper-skipping-files-and-rules-made-simple.md:16]: Haha, nice :heart: 

I didn't remember if for last 25 years, it justed randomly poped in my head.

File: packages/blog/data/2020/2020-12-10-new-in-symplify-9-skipper-skipping-files-and-rules-made-simple.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply packages/blog/data/2020/2020-12-10-new-in-symplify-9-skipper-skipping-files-and-rules-made-simple.md:7]: I see the missunderstanding now. 

For me the change happened already.
But for the reader, the change will happen after reading the post.

I'll use present time, thank you

File: packages/blog/data/2020/2020-12-10-new-in-symplify-9-skipper-skipping-files-and-rules-made-simple.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
+---
+id: 292
+title: "New in Symplify 9: Skipper - Skipping Files and Rules made Simple"
+perex: |
+    Symplify 9 brings **another component to easy life for package developers**. In Rector and ECS, you can ignore specific absolute paths or dynamic with `fnmatch()`. You can also ignore specific Rector, Fixer, and Sniff classes.
+    <br><br>
+    Both packages used almost the same syntax, yet there were differences based on which of them got fixed first. New Skipper component prevents this and allows **you to use standardized syntax in single `skip` parameters**.
+
+tweet: "New Post on #php 🐘 blog: New in #Symplify 9: Skipper -
```
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply packages/blog/data/2020/2020-12-10-new-in-symplify-9-skipper-skipping-files-and-rules-made-simple.md:11]: Guilty! ;)

Before Rector becoming a thing, I haven't used Twitter much, even for a months. So this helped my readers to use their own *rss*.

**Review by TomasVotruba — COMMENTED** [review]
Thank you Tomas :+1:

File: packages/blog/data/2020/2020-12-10-new-in-symplify-9-skipper-skipping-files-and-rules-made-simple.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (tomasnorre) [reply packages/blog/data/2020/2020-12-10-new-in-symplify-9-skipper-skipping-files-and-rules-made-simple.md:18]: I even remember the theme song still.. :)

**Comment by tomasnorre** [comment]
You're welcome.

**Comment by TomasVotruba** [comment]
It's out: https://tomasvotruba.com/blog/2020/12/10/new-in-symplify-9-skipper-skipping-files-and-rules-made-simple/

**Pull request merged**